blockchain/test/test.go

```diff
@@ -7,9 +7,9 @@ import (
 	"github.com/onsi/ginkgo"
 )
 
-// LocalContext allows you to mark a ginkgo context as being local-only.
+// Context allows you to mark a ginkgo context as being local-only.
 // It won't run if the CI environment variable is true.
-func LocalContext(description string, f func()) {
+func Context(description string, f func()) {
```

Comment:
Is it clear what this is when it's called as `test.Context`? Should it be renamed to something more descriptive? (e.g. `text.LocalContext`).